Nodejs/Product/Nodejs/SharedProject/CommonPropertyPage.cs

```diff
@@ -347,9 +32,9 @@ void IPropertyPage.SetObjects(uint count, object[] punk)
             {
                 if (punk[0] is ProjectConfig)
```

Comment:
Don't forget that you can now name the result of an `is` check.

Reply:
yep